var/spack/repos/builtin/packages/iegenlib/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# -----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
+# -----------------------------------------------------------------------------
+#   Note: This package was created by the ctop reaserch group at
+#         the University of Utah, Shool of Computing and is unifiliated
+#         with IEGenLib
+# -----------------------------------------------------------------------------
+from spack import *
+
+
+class Iegenlib(AutotoolsPackage):
+    """Inspector/Executor Generation Library for manipulating sets
+       and relations with uninterpreted function symbols. """
+
+    homepage = "http://github.com/CompOpt4Apps"
+    git      = "https://github.com/CompOpt4Apps/IEGenLib.git"
+
+    version('master', branch='master')
+
+    depends_on('cmake@2.6:', type='build')
+    depends_on('isl')
+    depends_on('texinfo', type='build')
```

Comment:
I assume this is for docs.  Can it be made optional?  Often, docs introduce a lot of dependencies that 90% of people don't want/need to build (because they get the docs already built off a website).

Reply:
I haven't been able to build it without texinfo.

Reply:
Really?  I can't find any texinfo in the Autotools build.  And the CMake build seems to work fine if `texi2pdf` is not available.  See:

https://github.com/CtopCsUtahEdu/chill/search?q=chill.texi&unscoped_q=chill.texi

Reply:
IEGenLib specifically